build/changelog.py

```diff
@@ -64,12 +128,14 @@ def main():
 
     args = parse_args()
     changelog = {}
-
     for commit_id in get_commit_ids(args.from_version, args.to_commit):
+        mention = ""
         commit_message = get_commit_message(commit_id)
-        group, line = get_changelog_message(commit_message, args.repo_url)
+        if not is_maintainer(commit_message):
+            mention = mention_author(commit_message, commit_id, args.token)
+        group, line = get_changelog_message(commit_message, mention, args.repo_url)
         changelog.setdefault(group, []).append(line)
-
+    
```

Comment:
Please remove extra whitespace (often there is an IDE/editor setting to automatically remove it)

Reply:
Got it, thanks!